index.html

```diff
@@ -3500,18 +3499,128 @@ <h3>
+                </h4>
+                <p>
 DID document metadata SHOULD include an <code>updated</code> property
 to indicate the timestamp of the last <a href="#update">Update operation</a>.
 This property MAY not be supported by a given <a>DID method</a>.
 The value of the property MUST follow the same formatting rules as the
 <code>created</code> property.
-                </dd>
-            </dl>
+                </p>
+              </section>
+
+            <section>
+              <h4>equivalentId</h4>
+                <p>
+                  A <a>DID Method</a> can define different forms of a DID that are 
+                  logically equivalent. An example is when a DID takes one form prior 
+                  to registration in a verifiable data registry and another form after 
+                  such registration. In this case, the <a>DID Method</a> specification 
+                  may need to express one or more DIDs that are logically equivalent to 
+                  the resolved DID as a property of the DID document. This is the 
+                  purpose of the <code><a>equivalentId</a></code>  property.
+                </p>
+        
+                <dl>
+                  <dt><dfn>equivalentId</dfn></dt>
+                  <dd>
+                    The value of <code>equivalentId</code> MUST be a 
+                    <a data-cite="INFRA#lists">list</a> where each item in the list 
+                    is a <a data-cite="INFRA#string">string</a> that conforms to the rules in Section <a href="#did-syntax"></a>.
+                  </dd>
+                  <dd>
+                    The relationship is a statement that each <code><a>equivalentId</a></code> 
+                    value is logically equivalent to the <code>id</code> property
+                    value and thus identifies the same DID subject.
+                  </dd>
+                  <dd>
+                    Each <code><a>equivalentId</a></code> DID value MUST be produced by, and a form of, 
+                    the same <a>DID Method</a> as the <code>id</code> property value. 
+                    (e.g. <code>did:example:abc</code> == <code>did:example:ABC</code>)
+                  </dd>
+                  <dd>
+                    A conforming <a>DID Method</a> specification MUST guarantee that each 
+                    <code><a>equivalentId</a></code> value is logically equivalent to the 
+                    <code>id</code> property value.
+                  </dd>
+                  <dd>
+                    A resolving party MUST retain the values from the <code>id</code> and 
+                    <code><a>equivalentId</a></code> properties to ensure any subsequent 
+                    interactions with any of the values they contain are correctly handled 
+                    as logically equivalent (e.g. retain all variants in a database so an 
+                    interaction with any one maps to the same underlying account).
```

Comment:
All of these statements are not testable -- we don't have conformance criteria for a "resolving party" in the spec. We probably don't want to slide down that slippery slope.

I'm fine w/ pulling in this PR as long as we remove the MUST here (and in each subsequent section).

We may not even need it -- given that this information is provided in the did document meta or resolver meta, the language probably belongs in those specifications. We can signal what we expect RPs to do here, but I don't think we can mandate any particular behavior.